hack/kiali/func-sm.sh

```
@@ -0,0 +1,369 @@
+
+  infomsg "Deleting the control plane and CNI namespaces"
+  for ns in ${doomed_namespaces}
+  do
+    ${OC} delete namespace ${ns}
+  done
+}
+
+status_servicemesh_operators() {
+  infomsg ""
+  infomsg "===== SERVICEMESH OPERATOR SUBSCRIPTION"
+  local sub_name="$(${OC} get subscriptions -n ${OLM_OPERATORS_NAMESPACE} -o name my-sailoperator)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)